src/server/models/Conversion.js

```diff
@@ -32,6 +33,31 @@ class Conversion {
 		return conn.none(sqlFile('conversion/create_conversions_table.sql'));
 	}
 
+	/**
+	 * Inserts standard conversions.
+	 * @param {*} conn The connection to use.
+	 */
+	static async insertStandardConversions(conn) {
+		// The table contains standard conversions' data.
+		// Each row contains: sourceName, destinationName, bidirectional, slope, intercept, note.
+		const standardConversions = [
+			['kWh', 'MJ', true, 3.6, 0, 'kWh → MJ'],
+			['MJ', 'M3_gas', true, 2.6e-2, 0, 'MJ → M3_gas'],
+			['MJ', 'BTU', true, 947.8, 0, 'MJ → BTU'],
+			['kg', 'Metric_ton', true, 1e-3, 0, 'kg → Metric ton'],
+			['Fahrenheit', 'Celsius', true, 1.8, 32, 'Fahrenheit → Celsius']
```

Comment:
I think the current one is correct since` Fahrenheit = 1.8 * Celsius + 32`. The order is similar to line 47, where `kg = 1e-3 * Metric_ton`.

Reply:
Maybe I have this wrong but I still have a concern. I think the correct conversion for line 47 is
kg = 1000 * Metric_ton
because there are 1000 kg in a metric ton. Thus, the conversion table is specifying the other way around of
Metric_ton = 1e-3 * kg
If that is true then the temperature conversion is backward.

Reply:
My bad. You are correct. The conversion should be reversed. Thank you for your explanation.